src/ocamlorg_web/lib/handler.ml

```diff
@@ -390,22 +380,33 @@ let package_docs t req =
   match version_opt with
   | None -> not_found req
   | Some version ->
-      let version = Ocamlorg_package.Version.to_string version in
-      let target = Ocamlorg_frontend.Url.package_doc package version in
+      let version_string =
+        if Ocamlorg_package.is_latest_version t name version then "latest"
+        else Ocamlorg_package.Version.to_string version
+      in
+      let target =
+        Ocamlorg_frontend.Url.package_doc package version_string
+          ~is_latest_url:false
+      in
       Dream.redirect req target
```

Comment:
this can be simplified, since all this function needs to do is to redirect to the latest-URL of the package documentation.

Reply:
It doesn't need to just redirect to latest, its supposed to redirect to latest if the user wants to go to latest OR versioned if thats what the user wants, thats why there's the if then else statement